helm/flink-operator/templates/rbac.yaml

```diff
@@ -132,7 +190,25 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 subjects:
   - kind: ServiceAccount
-    name: {{ template "flink-operator.serviceAccountName" . }}
+    name: {{ include "flink-operator.serviceAccountName" . }}
+    namespace: {{ .Values.operatorNamespace.name }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Given that watchNamespaces is not configured, I am thinking whether we really need to create the `ClusterRole` and `ClusterRoleBinding` for flink job. Maybe `Role` and `RoleBinding` is enough. If users want to create a FlinkDeployment in a different namespace with flink-k8s-operator, then they should create the `serviceAccount` `Role` `RoleBinding` manually.

Reply:
Fair point, given that we are trying to limit the footprint of the roles created I will create only role/rolebing in the operator namespace.